CHANGELOG.md

```diff
@@ -15,6 +15,10 @@
 
 ##### Enhancements
 
+* Windows support - do not use fork when updating the index  
```

Comment:
This was in the wrong spot. Updated to reflect correct release which was 1.7.0.beta.1